kalite/distributed/static/js/distributed/search/views.js

```diff
@@ -116,6 +116,22 @@ window.AutoCompleteView = BaseView.extend({
             return compvalue;
         });
 
+        // Then sort again by availability!
+        // Use the original ordering to make Array.prototype.sort stable
+        ids_with_ind = _.map(ids_filtered, function(id, index) {
```

Comment:
Identifier 'ids_with_ind' is not in camel case.<br>Identifier 'ids_filtered' is not in camel case.